### PR TITLE
wrote and styled Hero, back to top button is functional

### PR DESCRIPTION
**Pull request by shane-jeon**
- Added all elements required for Hero 
- back to top button is functional (disregarded "(not displaying on a mobile screen)" for now)

Screen Shot 2023-04-06 at 2 26 03 PM
Screen Shot 2023-04-06 at 2 26 12 PM

### CODE REVIEW

File: pages/index.tsx
Comment (najwanan) [inline pages/index.tsx:1]: Let's uncomment out your component for the final. :-)

File: components/Hero.jsx
Comment (najwanan) [inline components/Hero.jsx:1]: Pretty good. I do have to request a  refactor of this code to incorporate tailwind for uniformity in the codebase. You'll want to put all of your styling within the className for each element. When you refactor, make sure that the hero covers the width of the screen.

**Review by najwanan — CHANGES_REQUESTED** [review]
Solid start but needs some refactoring. Lmk if you want to talk through any changes.

**Comment by shane-jeon** [comment]
Changed styling to consider responsivity. Checked for all screen sizes.
fullBrowser
decreasingBrowserSize
mostDecreasedBrowserSize
tablet
mobile

**Comment by najwanan** [comment]
@shane-jeon the text is still a bit off and not scaling properly. We can pick this up in our 1:1

**Review by najwanan — CHANGES_REQUESTED** [review]
It looks like some of your other work made it into this branch, and it's currently not rendering properly. 

Let's remove all the "Get Involved" section work. 

Also looks like it's scaling better for devices. Still has some issues with scaling for smaller devices. But let's merge in and we can create a separate ticket to fix that later on.

File: pages/index.tsx [hunk 1]
```diff
@@ -1,88 +1,92 @@
 import Head from "next/head";
 import Image from "next/image";
+import Hero from "./Hero";
 import Navbar from "./Navbar";
+import GetInvolved from "./GetInvolved";
```
Comment (najwanan) [inline pages/index.tsx:5]: Can we remove this please?

File: pages/index.tsx [hunk 2]
```diff
@@ -1,88 +1,92 @@
...
 
 export default function Home() {
   return (
     <>
       <Navbar />
+      <Hero />
+      <GetInvolved />
```
Comment (najwanan) [inline pages/index.tsx:12]: And remove this one as well please? It should render okay afterwards.

Comment (shane-jeon) [reply pages/index.tsx]: removed both

**Review by najwanan — APPROVED** [review]

**Comment by najwanan** [comment]
Ready to merge @shane-jeon

**Pull request merged**